src/NetscriptFunctions/Grafting.ts

```diff
@@ -52,6 +52,18 @@ export function NetscriptGrafting(): InternalAPI<IGrafting> {
       return graftableAugs;
     },
 
+    isGrafting: (ctx) => () => {
+      checkGraftingAPIAccess(ctx);
+      let grafting = false;
+      const work = Player.currentWork;
```

Comment:
You can simplify and consolidate the conditional logic like this.

```suggestion
    const work = Player.currentWork;
    return work && work.type === "GRAFTING";
```

Reply:
While that simplifies it, it does give more information away to the player.  They could use it to tell if the player is just busy, and not just grafting (something only available in the singularity API)
isGrafting is null when not doing any work
isGrafting is false when doing work that isn't grafting 
isGrafting is true when Grafting
The compiler also didn't like the type of boolean, as boolean can't be null.  Had to change it to an 'any' type.  No idea if that's a bad thing.

Reply:
I've changed the work declaration line to return false if null
`const work = Player.currentWork ?? false;`
Thanks for the suggestion